cmd/describe_affected_test.go

```
@@ -0,0 +1,131 @@
+			fs.Bool("skip", false, "Skip")
+			fs.String("pager", "", "Pager")
+			fs.String("stack", "", "Stack")
+			fs.String("format", "", "Format")
+			fs.String("file", "", "Output file")
+			fs.String("query", "", "Query")
+
+			// Set flags as specified in the test case
+			tt.setFlags(fs)
+
+			// Call the function
+			if tt.expectedPanic {
```

golangci-lint (1.64.8)

[warning] `if tt.expectedPanic` has complex nested blocks (complexity: 4)

(nestif)